### ISSUE TITLE
Release of pretrained decoder for RVM

### ISSUE THREAD

**Issue by kris-singh** [google-deepmind/representations4d#4]
Hi, 

Could you also please release the trained decoder for the RVM model? 
The decoder would help us better investigate the paper's findings and ultimately extend the work. 

Thanks,
Krish Singh

**Comment by yangyi02** [comment]
Hi @kris-singh,

Thanks for the request! We have now released the decoder in our [Colab notebook](https://colab.sandbox.google.com/github/google-deepmind/representations4d/blob/main/colabs/rvm_inference_demo.ipynb).

You can check it out under the `Masked Reconstruction Demo` section.
